src/crc.jl

```diff
@@ -0,0 +1,87 @@
+module CRC
```

Comment:
wat?
Why?

Also we have CRC in the stdlibs
https://github.com/JuliaLang/julia/blob/v1.1.0/stdlib/CRC32c/src/CRC32c.jl

Reply:
Oh I didn't know. Does it give exactly the same output as the implementation I have?

Reply:
Not sure,test it?